front-end-side/src/Pages/Data.js

```diff
@@ -155,20 +155,21 @@ const EnterData = () => {
                                             )}
                                         </tbody>
 
-                                        <tfoot>
-                                            <tr>
-                                                <td colSpan="3">Spent in total</td>
-                                                <td colSpan="3">{parseFloat(newExpenses).toFixed(2)}</td>
-                                            </tr>
-                                        </tfoot>
-                                    </table>
-
-                                    {allShown ?
-                                        <button type="button" onClick={showLess}>Show less</button>
-                                        : <button type="button" onClick={showAll}>Show more</button>
+                                    <tfoot>
+                                        <tr>
+                                            <td colSpan="3">Spent in total</td>
+                                            <td colSpan="3">{parseFloat(newExpenses).toFixed(2)}</td>
+                                        </tr>
+                                    </tfoot>
+                                </table>
+                                    {
+                                        fileData.data.length <= 5 ? null :
```

Comment:
Why you chose number 5?

Reply:
It just looks better when we use 5, because when the number is higher, we need to scroll down a little bit

Reply:
sounds right